repository/flink/docs/demo/financial-fraud/README.md

```diff
@@ -42,50 +42,25 @@ $ minikube start --vm-driver=hyperkit --cpus=6 --memory=9216 --disk-size=10g
 
 ### Install KUDO and the KUDO CLI
 
-- Have current KUDO CLI `v0.10.x` [installed](https://kudo.dev/docs/cli.html#setup-the-kudo-kubectl-plugin)
+- Have current KUDO CLI `v0.15.x` [installed](https://kudo.dev/docs/cli.html#setup-the-kudo-kubectl-plugin)
     - Run `kubectl kudo version`, the output should look like:
         ```bash
         $ kubectl kudo version
         KUDO Version: version.Info{GitVersion:"0.10.0", GitCommit:"d2310b12", BuildDate:"2020-01-10T18:54:59Z", GoVersion:"go1.13.6", Compiler:"gc", Platform:"darwin/amd64"}
```

Comment:
the output example still shows `0.10.0`.  It makes sense that we don't have the build date or sha as we haven't had a release for this version... but it doesn't make sense to change the version from `v0.10.x` to `v0.15.x` and not change the gitVersion to `"0.15.0"`

Reply:
Good catch. I missed it